packages/react/src/components/Tab/Tab.js

```diff
@@ -177,7 +184,8 @@ export default class Tab extends React.Component {
           onKeyDown(evt);
         }}
         role="presentation"
-        selected={selected}>
+        selected={selected}
+        aria-controls={`${id}__panel`}>
```

Comment:
Should this be on the `a` that has `role="tab"`?

Reply:
Makes sense, fixed.